restaurants/src/app/services/restaurants.service.ts

```diff
@@ -34,8 +35,13 @@ export class RestaurantsService {
       var id: string = uuidv4(); 
       this.RESTAURANTS[id] = newRestaurant;
     }
-  
+    
     delete(id: string){
       delete this.RESTAURANTS[id];
     }
+
+    getRestaurant(id: string): Observable<Restaurant> {
+        const restaurant = this.RESTAURANTS[id];
+        return of(restaurant);
```

Comment:
can probably simplify lines 44 and 45 to a single line - not really necessary to declare a new variable if it only gets used once (sometimes it's good to for readability, but in this case the code is still very short)

```
return of(this.RESTAURANTS[id];
```

Reply:
done